sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -524,6 +535,8 @@ case class RegExpExtract(subject: Expression, regexp: Expression, idx: Expressio
         $termPattern.matcher($subject.toString());
       if ($matcher.find()) {
         java.util.regex.MatchResult $matchResult = $matcher.toMatchResult();
+        org.apache.spark.sql.catalyst.expressions.RegExpExtract
```

Comment:
don't hard code the class name. `${classOf[RegExpExtract].getName}. checkGroupIndex`

Reply:
OK